lib/persist.dart

```diff
@@ -52,7 +50,9 @@ BasicSettings _loadBasic(SharedPreferences prefs) {
           altitude: prefs.getDouble('altitude') ?? defaultAltitude));
 
   basicInput.waz = prefs.getDouble('waz') ?? 0;
```

Comment:
wtf does "waz" stand for?

Reply:
Refer to ST-X-3 Appendix II; wind azimuth in degrees; (https://cfs.nrcan.gc.ca/publications/download-pdf/10068)

This trips up @conbrad as well all the time. I've opted to stick to the conventions used in the wildfire papers, and in the CFFDRS R package. Using descriptive variable names ends up being more confusing, because it's hard to trace back the math.